### ISSUE TITLE
CDATA section ignored in KML description

### ISSUE THREAD

**Issue by robertico01** [mapbox/togeojson#56]
I've noticed some issues parsing CDATA sections in a KML file

This doesn't work;
```
      <description>
        <![CDATA[
             Here is some text
        ]]>
      </description>
```
but this does;
```
      <description><![CDATA[Here is some text]]></description>
```
and this also;
```
      <description><![CDATA[

             Here is some text
      ]]></description>
```
As far as I know they're all valid. Nevertheless a bug?

**Comment by tmcw** [comment]
I'll certainly work on this when I have time: when I don't have time, I'm not ignoring it, I'm working on (many) other things, so please summon either patience or the time to help fix it.

**Comment by tmcw** [comment]
Fixed in #68
